nautobot/dcim/tests/test_filters.py

```diff
@@ -284,14 +284,16 @@ def common_test_data(cls):
     Prefix.objects.create(prefix=netaddr.IPNetwork("192.168.2.0/24"), location=loc1, status=prefix_status)
 
     # TODO: remove these once we have a Sites fixture; for now SiteTestCase needs VLANGroups and VLANs with Sites
```

Comment:
Comment is probably obsolete now; not blocking but would be interesting to see whether the below can be removed now that we have appropriate factories in place for VLANGroup and VLAN.

Reply:
Yup, It can be removed but it causes test failures where `VLAN` and `VLANGroup` location are required to be one of the above locations